tabs-tabs-tabs/tabs.js

```diff
@@ -12,6 +12,30 @@ function firstUnpinnedTab(tabs) {
   }
 }
 
+function listTabs() {
+  chrome.tabs.query({
+    currentWindow: true
+  }, function(tabs) {
+    var tabsList = document.getElementById('tabs-list'),
+        currentTabs = document.createDocumentFragment();
```

Comment:
I don't think we enforce this in a style guide, but could you make this `var currentTabs` please.